### DISCUSSION TITLE
Why can I no longer uninstall extensions I don't want?

### DISCUSSION THREAD

**Discussion by periodic1236** [standardnotes/forum#2767] [General]
*Have you read the [Longevity Statement](https://standardnotes.org/longevity)?*
Yes

*Is your feature request related to a problem? Please describe.*
Before version 3.9.0, I could avoid having a cluttered interface on desktop by only keeping a minimal set of editors and themes installed. If at any time I needed additional functionality, I could browse the available extensions and reinstall the one I wanted with a few simple clicks.

*Describe the solution you'd like*
Bring back the extension storefront and allow me to manage the installed extensions. The new preferences pane is hard to navigate compared to before. I feel as if you've added complexity to the UX instead of making it simpler.

*Describe alternatives you've considered*
I've looked in the app, FAQ/help sections, blog, recent issues, and release notes, and cannot find anything that explains this change or anything to do with managing extensions.

**Comment by atmoio** [comment]
The current UI/UX is in a transitionary state. In updates planned for the near future, you won't have a cluttered editor or theme list even if you have a hundred editors. Please bare with us for now.

**Comment by cyb3rz3us** [comment]
After nearly 4 months, has the "transitionary state" been resolved?  I too am wondering where the ability to easily manage all extensions has gone...

**Comment by atmoio** [comment]
> has the "transitionary state" been resolved

It has. You can delete extensions via Preferences > General > Advanced.

**Comment by cyb3rz3us** [comment]
Delete perhaps - but where is the ability to browse the extensions pool?  As the OP mentioned, this was a very useful feature...

**Comment by atmoio** [comment]
There is no concept of extension pool anymore. We treat these as just features in the app. So some are in the editor menu and some are in the themes menu. The closest we have to a gallery is https://standardnotes.com/features but that doesn't yet drill too much into each one.

**Comment by cyb3rz3us** [comment]
First - "no concept of extension pool anymore" = **bad idea**.  Especially without (apparently) first reaching out to your user base nor providing a functional alternative.  Next, I seem to recall that there were far more extensions than currently exist as "Features" - should this be interpreted as the other extensions are no longer available?  Also, what is SN's plan WRT users being able to access some type of extension library?  Do you envision SN hosting this?  Or maybe other users setup their own libraries to possibly share with others?  Something else altogether?
